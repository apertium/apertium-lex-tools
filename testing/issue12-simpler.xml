<rules>
  <rule weight="2.0">                    <!-- rule1 -->
    <or>
      <match tags="seen"/>               <!-- A -->
      <match tags="unseen"/>             <!-- B -->
    </or>
    <match lemma="X" tags="adv"> <!-- C -->
      <select lemma="X"/>
    </match>
  </rule>

  <rule weight="2.0">                    <!-- rule2 -->
    <match tags="unseen"/>               <!-- B -->
    <match lemma="Y" tags="n">
      <select lemma="Y"/>
    </match>
  </rule>
</rules>
