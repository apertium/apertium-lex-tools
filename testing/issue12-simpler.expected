^context<seen>/context<seen>$ ^Y<n>/Y<n>/want<n>$
